src/Admin/BaseFieldDescription.php

```diff
@@ -418,40 +419,37 @@ public function getFieldValue($object, $fieldName)
             return $this->getFieldValue($child, substr($fieldName, $dotPos + 1));
         }
 
-        // prefer method name given in the code option
+        // NEXT_MAJOR: Remove this code.
         if ($this->getOption('code')) {
-            $getter = $this->getOption('code');
-
-            if (!method_exists($object, $getter)) {
-                @trigger_error(
-                    'Passing a non-existing method in the "code" option is deprecated'
-                    .' since sonata-project/admin-bundle 3.x and will throw an exception in 4.0.',
-                    \E_USER_DEPRECATED
-                );
+            @trigger_error(
```

Comment:
Since `code` is always set in the persistence bundles, this is always triggered: https://github.com/sonata-project/SonataDoctrineORMAdminBundle/runs/1903176971?check_suite_focus=true#step:8:108

Reply:
I think we should remove the code in the persistence bundle.

If the name is `getfoo` and the `fieldName` is `bar`, because we're passing the `code`, the value will be `$object->getFoo()` instead of relying on `bar` ; this seems to be a bug to me.

Reply:
:thumbsup: